task02-pixel-perfect/styles.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: 'Roboto Slab';
+}
+
+h1 {
+  margin-bottom: auto;
+}
+
+.card {
+  position: relative;
+
+  width: 220px;
+  height: 280px;
+  border-radius: 20px;
+  background-image: linear-gradient(rgba(255,255,255,0), rgb(0,0,0));
+  background-position: bottom;
+  background-repeat: no-repeat;
+  background-size: 100% 40%;
+
+  overflow: hidden;
+}
+
+.card__img {
+  position: relative;
+  z-index: -1;
+
+  width: 100%;
+  height: 100%;
+
+  object-fit: cover;
+}
+
+.card__name {
+  position: absolute;
+  bottom: 30px;
+
+  width: 100%;
+
+  font-size: 22px;
+  font-weight: bold;
+  text-align: center;
+  color: white;
+}
+
+.characters-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  margin-left: 170px;
+  margin-right: 170px;
```

Comment:
Please combine this margins into one, margin can be used into a single line of code and specify a value for each directions (margin: top, right, bottom, left). For this case you can use (margin: 0 170px) which indicates the vertical and horizontal margins